### PR TITLE
add controlled ve curves for S>=s

### PR DESCRIPTION
**Pull request by youyifong**

### CODE REVIEW

**Comment by benkeser** [comment]
@youyifong, the tex live issue that was breaking travis builds has been fixed; could you merge your branch with `master` and push to trigger a new build?

**Comment by youyifong** [comment]
Rebuilding

File: cor_coxph/code/cor_coxph.R [hunk 1]
```diff
@@ -79,13 +81,13 @@ t0=max(dat.vacc.pop[dat.vacc.pop[["EventIndPrimaryD"%.%pop]]==1, "EventTimePrima
...
 # trial-specific formula
 form.s = as.formula(paste0("Surv(EventTimePrimaryD",pop,", EventIndPrimaryD",pop,") ~ 1"))
-if (study_name == "mock") {
-    form.0 =            update (form.s, ~.+ MinorityInd + HighRiskInd + Age) #  Age is to be replaced by BRiskScore
-    form.0.logistic = as.formula(paste0("EventIndPrimaryD",pop,"  ~ MinorityInd + HighRiskInd + Age"))  #  Age is to be replaced by BRiskScore
-} else if (study_name == "moderna") {
-    form.0 =            update (form.s, ~.+ MinorityInd + HighRiskInd + BRiskScore)
-    form.0.logistic = as.formula(paste0("EventIndPrimaryD",pop,"  ~ MinorityInd + HighRiskInd + BRiskScore"))
-} else stop("")
+if (endsWith(data_name, "riskscore.csv")) {
```
Comment (benkeser) [inline cor_coxph/code/cor_coxph.R:84]: should this be 
```r
if (endsWith(data_name_updated, "riskscore.csv")) {
```
?

**Comment by youyifong** [comment]
Good catch, thanks.

**Pull request merged**